wormhole-connect/src/config/types.ts

```diff
@@ -76,6 +76,8 @@ export type SupportedRoutes = keyof typeof Route;
 
 export type Network = 'mainnet' | 'testnet' | 'devnet';
 
+export { WormholeConnectEvent } from 'telemetry/types';
```

Comment:
seems weird to export it from `config/types`, and have index import it from here. why can't index just import it directly from `telemetry/types`?

Reply:
oh yes, you right, i just refactor the export code